src/main/templates/net/imagej/ops/Ops.list

```diff
@@ -197,6 +197,7 @@ namespaces = ```
 		[name: "histogram",                      iface: "Histogram"],
 		[name: "integral",                       iface: "Integral"],
 		[name: "invert",                         iface: "Invert"],
+		[name: "invertBigTypes",                 iface: "InvertBigTypes"],
```

Comment:
We need to make `invert` work for both "big" and "small" types, without a separate op.